rest/route/task_generate.go

```diff
@@ -47,6 +50,23 @@ func (h *generateHandler) Run(ctx context.Context) gimlet.Responder {
 	if err := data.GenerateTasks(h.taskID, h.files); err != nil {
 		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "generating tasks for task '%s'", h.taskID))
 	}
+	t, err := task.FindOneId(h.taskID)
+	if err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "getting task '%s'", h.taskID))
+	}
+	if t == nil {
+		return gimlet.MakeJSONErrorResponder(errors.Errorf("task '%s' not found", h.taskID))
+	}
+	ts := utility.RoundPartOfMinute(1).Format(units.TSFormat)
+	j, err := units.CreateAndEnqueueGenerateTasks(ctx, evergreen.GetEnvironment(), *t, ts)
+	if err != nil {
```

Comment:
nit: you can omit the `if err != nil` since `message.WrapError` is smart enough to suppress the log when it's a nil error.